backend/src/main/kotlin/com/tngtech/apicenter/backend/config/SecurityConfiguration.kt

```diff
@@ -25,12 +26,33 @@ class SecurityConfiguration : WebSecurityConfigurerAdapter() {
             .csrf().disable()
             .authorizeRequests()
             .antMatchers("/sessions").permitAll()
-            .and()
-            .authorizeRequests()
+
+            .antMatchers("/").permitAll()
+            .antMatchers("/3rdpartylicenses.txt").permitAll()
+            .antMatchers("/favicon.ico").permitAll()
+            .antMatchers("/index.html").permitAll()
+            .antMatchers("/main.js").permitAll()
+                .antMatchers("/main.js.map").permitAll()
+            .antMatchers("/open-iconic.*").permitAll()
+            .antMatchers("/polyfills.js").permitAll()
+                .antMatchers("/polyfills.js.map").permitAll()
+            .antMatchers("/runtime.js").permitAll()
+                .antMatchers("/runtime.js.map").permitAll()
+            .antMatchers("/styles.js").permitAll()
+                .antMatchers("/styles.js.map").permitAll()
+            .antMatchers("/scripts.js").permitAll()
+                .antMatchers("/scripts.js.map").permitAll()
+            .antMatchers("/vendor.js").permitAll()
+                .antMatchers("/vendor.js.map").permitAll()
             .anyRequest().authenticated()
```

Comment:
What would happen if we did not define those expections?

Reply:
Anyone trying to use ApiCenter's frontend eg. access the index page would get a 403 error, because the backend is not otherwise going to serve unauthorised users any static files.

Reply:
Are they then prompted to log in, or does it just fail?
If the exceptions are necessary, then I'd suggest to plainly allow `*.js*` and `*.js.map`